test/box-luatest/upgrade/1.6.8/gen.lua

```diff
@@ -0,0 +1,61 @@
+--
+-- This script creates a snapshot on Tarantool 1.6.9-94-ga5da60bdd.
```

Comment:
But the folder is called `upgrade/1.6.8`. Shouldn't it use 1.6.8 then?

Reply:
Renamed the dir name to 1.6.9)